### PR TITLE
add way to set size of ICE pool

### PR DESCRIPTION
**Pull request by fluffy**
Fixes https://github.com/w3c/webrtc-pc/issues/153

### CODE REVIEW

**Comment by martinthomson** [comment]
LGTM

**Comment by fluffy** [comment]
Justin raised point that this needs to be mentioned in the setConfiguration to later change it, as well as in in inial

**Comment by fluffy** [comment]
I think this has all the comments from the interm meeting in Seattle and is ready to merge

File: webrtc.html [hunk 1]
```diff
@@ -1419,6 +1427,13 @@
               </li>
 
               <li>
+              <p>Let the value of <var>configuration</var>'s
+                iceCandidatePoolSize member be the User Agent's prefetched ICE
```
Comment (martinthomson) [inline webrtc.html]: Nit: link for iceCandidatePoolSize

**Comment by martinthomson** [comment]
Also LGTM

**Comment by alvestrand** [comment]
Met with approval in seattle (but needs update and review). Should be merged.

**Pull request merged**